Packs/FiltersAndTransformers/Scripts/IfElif/IfElif.py

```diff
@@ -0,0 +1,122 @@
+import demistomock as demisto
+from CommonServerPython import *
+from collections.abc import Callable
+from functools import reduce, partial
+import ast
+
+
+class IfElif:
+    operator_functions: dict[type, Callable] = {
+        # comparison operators:
+        ast.Eq: lambda x, y: x == y,
+        ast.NotEq: lambda x, y: x != y,
+        ast.Lt: lambda x, y: x < y,
+        ast.LtE: lambda x, y: x <= y,
+        ast.Gt: lambda x, y: x > y,
+        ast.GtE: lambda x, y: x >= y,
+        ast.In: lambda x, y: x in y,
+        ast.NotIn: lambda x, y: x not in y,
+        # boolean operators:
+        ast.And: lambda x, y: x and y,
+        ast.Or: lambda x, y: x or y,
+        # unary operators:
+        ast.Not: lambda x: not x,
+        ast.USub: lambda x: -x,
+    }
+
+    def __init__(self, value, conditions, flags=None):
+        self.conditions: list
+        self.handle_flags(argToList(flags))
+        self.load_conditions_with_values(conditions, value)
+
+    def handle_flags(self, flags: list):
+        self.regex_flags = (
+            re.DOTALL * ('regex_dot_all' in flags)
+            | re.MULTILINE * ('regex_multiline' in flags)
+            | re.IGNORECASE * ('case_insensitive' in flags)
+        )
+        self.functions = {
+            'regex_match': partial(
+                re.fullmatch if 'regex_full_match' in flags else re.search,
+                flags=self.regex_flags
+            )
+        }
+        if 'case_insensitive' in flags:
+            def eq(x, y):
+                return repr(x).lower() == repr(y).lower()
+            self.operator_functions |= {
+                ast.Eq: eq,
+                ast.NotEq: lambda x, y: not eq(x, y),
+            }
+
+    def load_conditions_with_values(self, conditions, values):
+        def from_value(keys: re.Match):
+            return repr(demisto.dt(values, keys[1]))
+        conditions = re.compile('#{([\s\S]+?)}').sub(from_value, conditions)
```

Comment:
we need to replace the conditions with what we got from the values after the tree is parsed, otherwise we will have a risk from injection